packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -329,6 +329,17 @@ class MuteState {
     );
   }
 
+  /**
+   * Returns true if the user is locally muted
+   *
+   * @public
+   * @memberof MuteState
+   * @returns {Boolean}
+   */
+  public isLocallyMuted() {
```

Comment:
Looks good to me as we are going have only one state for server or local 

@marcin-bazyl  any thoughts on this

Reply:
looks good, when we send Roap messages to Locus we need to indicate only the local mute, not the remote one.

Reply:
ok so if someone does any action and remote is muted then it will change the local to muted also 

@mkesavan13  can you check the media request what values it is using

Reply:
Noted @arun3528 & @marcin-bazyl 

Will keep an eye for this when we move to webrtc-core lib. Thank you.